ui/src/components/Layout/Layout.tsx

```diff
@@ -0,0 +1,31 @@
+import {useState} from 'react';
+import {Sidebar} from '../../components/Sidebar/Sidebar';
+import {TopBar} from '../../components/TopBar';
+import styles from './styles.module.scss';
+
+interface LayoutProps {
+  children: React.ReactNode;
+}
+
+export const Layout: React.FC<LayoutProps> = props => {
+  const [isSidebarOpenOnMobile, setIsSidebarOpenOnMobile] = useState(false);
+  return (
+    <div className={styles.container}>
+      <Sidebar
+        isSidebarOpenOnMobile={isSidebarOpenOnMobile}
+        setIsSidebarOpenOnMobile={setIsSidebarOpenOnMobile}
+      />
+      <div
+        className={`${styles.mainContent} ${
+          isSidebarOpenOnMobile ? styles.mainContentBlur : ''
+        }`}
```

Comment:
nit: [classnames](https://www.npmjs.com/package/classnames) could be used to have conditional classes cleanly.

Maybe create an issue for this.